pkg/resource/table/custom_update_api.go

```diff
@@ -0,0 +1,27 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package table
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	svcsdk "github.com/aws/aws-sdk-go/service/dynamodb"
+)
+
+var (
+	defaultBillingMode = svcsdk.BillingModeProvisioned
+
+	defaultStreamSpeficication = &svcsdk.StreamSpecification{
```

Comment:
what is a Speficication? :)

Reply:
Also, any reason to place this into `custom_update_api.go` since this actually doesn't have to do with the update code paths (it's the post-create code paths, not update). What about calling the file `defaults.go`?

Reply:
++ for `defaults.go`

Since we're squashing commits i didn't want to push defaults and update operations support for this PR - i'll do it in separe one.